contract-tests/src/bin/sse-test-api/stream_entity.rs

```diff
@@ -89,8 +87,8 @@ impl Inner {
         true
     }
 
-    fn build_client(config: &Config) -> Result<es::Client<Connector>, String> {
-        let mut client_builder = match es::Client::for_url(&config.stream_url) {
+    fn build_client(config: &Config) -> Result<Pin<Box<dyn es::Client>>, String> {
+        let mut client_builder = match es::for_url(&config.stream_url) {
```

Comment:
`for_url` is no longer implemented on `Client`, since `Client` is no longer a struct.